src/index.js

```diff
@@ -1,31 +1,40 @@
-/* eslint-disable no-unused-vars */
-/* eslint-disable import/no-extraneous-dependencies */
+import '@babel/polyfill';
+import '@babel/register';
 import express from 'express';
+import morgan from 'morgan';
 import dotenv from 'dotenv';
 import swaggerJSDoc from 'swagger-jsdoc';
 import swaggerUi from 'swagger-ui-express';
 import bodyParser from 'body-parser';
 import blogRoutes from './routes/index.js';
+import sequelize from './database/config/db.js';
+
 import userRoutes from './routes/user.route.js';
 import options from './docs/apidoc.js';
-import redis from './helpers/redis';
 
 dotenv.config();
 const app = express();
+
+app.use(morgan('tiny'));
+
 const { PORT } = process.env;
 
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 
 const specs = swaggerJSDoc(options);
+app.use('/myapi', swaggerUi.serve, swaggerUi.setup(specs));
+
+sequelize.sync();
 
 app.use('/api-doc', swaggerUi.serve, swaggerUi.setup(specs));
 app.use('', blogRoutes);
```

Comment:
You can  remove this blogroute cause we won't use it anymore.

Reply:
And as earlier requested we changed swagger route from /myapi  to /api-docs

Reply:
> You can remove this blogroute cause we won't use it anymore.



> And as earlier requested we changed swagger route from /myapi to /api-docs

@ishimwe5555 ✅ changed `swagger route from /myapi to /api-docs`
@Pacific911 ✅ removed the blogroute